curiosity_rover/Dockerfile.GUI

```diff
@@ -23,9 +22,9 @@ RUN wget https://packages.osrfoundation.org/gazebo.gpg -O /usr/share/keyrings/pk
 RUN apt-get update -qq \
     && apt-get install -y \
     gz-${GZ_VERSION} \
-    libgz-sim7 \
-    libgz-transport12 \
-    libgz-gui7 \
+    libgz-sim8 \
```

Comment:
dont think we should have this. gz-harmonic will pull the right versions of everything